.github/workflows/ci.yaml

```diff
@@ -1,20 +1,513 @@
+        apiVersion: kind.x-k8s.io/v1alpha4
+        containerdConfigPatches:
+        - |-
+          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.local:5000"]
+            endpoint = ["http://registry.local:5000"]
+        EOF
+
+    - name: Create Kind Cluster
+      uses: helm/kind-action@v1.8.0
+      with:
+        cluster_name: e2e
+        config: kind.yaml
+
+    - name: Docker Login
+      uses: docker/login-action@v2.2.0
+      with:
+        registry: ${{ env.REGISTRY_URL }}
+        username: ${{ env.REGISTRY_USER }}
+        password: ${{ env.REGISTRY_PASS }}
+
+    - name: Run tests
+      run: |
+        # make the registry container accessible by name from inside the cluster
+        docker network connect kind registry.local
+
+        kapp deploy -a kpack -y -f prerelease.yaml
+
+        export IMAGE_REGISTRY=${{ env.REGISTRY_URL }}
+        export IMAGE_REGISTRY_USERNAME=${{ env.REGISTRY_USER }}
+        export IMAGE_REGISTRY_PASSWORD=${{ env.REGISTRY_PASS }}
+
+        export GIT_PRIVATE_REPO=${{ vars.E2E_PRIVATE_REPO }}
+        export GIT_BASIC_USERNAME="${{ secrets.E2E_PRIVATE_REPO_USERNAME }}"
+        export GIT_BASIC_PASSWORD="${{ secrets.E2E_PRIVATE_REPO_PASSWORD }}"
+        export GIT_SSH_PRIVATE_KEY="${{ secrets.E2E_PRIVATE_REPO_PRIVATE_KEY }}"
+
+        make e2e
+        kapp delete -a kpack -y
+
+    - name: Run tests with Istio
+      run: |
+        helm repo add istio https://istio-release.storage.googleapis.com/charts
+        helm repo update
+        kubectl create namespace istio-system
+        helm install istio-base istio/base -n istio-system --wait
+        helm install istiod istio/istiod -n istio-system --wait
+
+        cat <<EOF > overlay.yaml
+        #@ load("@ytt:overlay", "overlay")
+        #@overlay/match by=overlay.subset({"metadata":{"name":"kpack-controller"}, "kind": "Deployment"})
+        ---
+        spec:
+          template:
+            spec:
+              containers:
+              #@overlay/match by="name"
+              - name: controller
+                #@overlay/match-child-defaults missing_ok=True
+                env:
+                #@overlay/match by="name"
+                #@overlay/replace or_add=True
+                - name: INJECTED_SIDECAR_SUPPORT
+                  value: "true"
+        EOF
+
+        ytt -f prerelease.yaml -f overlay.yaml | kapp deploy -a kpack -y -f-
+
+        export IMAGE_REGISTRY=${{ env.REGISTRY_URL }}
+        export IMAGE_REGISTRY_USERNAME=${{ env.REGISTRY_USER }}
+        export IMAGE_REGISTRY_PASSWORD=${{ env.REGISTRY_PASS }}
+        export KPACK_TEST_NAMESPACE_LABELS="istio-injection=enabled"
+
+        export GIT_PRIVATE_REPO=${{ vars.E2E_PRIVATE_REPO }}
+        export GIT_BASIC_USERNAME="${{ secrets.E2E_PRIVATE_REPO_USERNAME }}"
+        export GIT_BASIC_PASSWORD="${{ secrets.E2E_PRIVATE_REPO_PASSWORD }}"
+        export GIT_SSH_PRIVATE_KEY="${{ secrets.E2E_PRIVATE_REPO_PRIVATE_KEY }}"
+
+        make e2e
```

Comment:
This test isn't needed as Istio is [0.8.0](https://github.com/buildpacks-community/kpack/releases/tag/v0.8.0) onwards

Reply:
removed